frontend/src/assets/css/color-variables.css

```diff
@@ -0,0 +1,29 @@
+:root {
+    /* Background */
+    --background-blue-600: #163676;
+    --background-blue-300: #98B2DE;
+    --background-blue-200: #D7DFFE;
+    --background-blue-100: #EFF3FF;
+
+    /* Font */
+    --text-color-gray-400: #130138;
+    --text-color-gray-300: #5A6474;
+    --text-color-gray-200: #9AACBD;
+    --text-color-white-100: #FFFFFF;
+
+    /* Brand */
+    --brand-color-turquoise: linear-gradient(95.77deg, #00D7BD -14.06%, #03BFD9 101.51%);
```

Comment:
```suggestion
    --gradient-turquoise: linear-gradient(95.77deg, #00D7BD -14.06%, #03BFD9 101.51%);
```

Reply:
Done in commit [03db74d](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/03db74d7ec35888e47792a6337700fc5e0d1a4a5)